CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * [CHANGE] The frontend component has been refactored to be easier to re-use. When upgrading the frontend, cache entries will be discarded and re-created with the new protobuf schema. #1734
 * [CHANGE] Remove direct DB/API access from the ruler
 * [CHANGE] Removed `Delta` encoding. Any old chunks with `Delta` encoding cannot be read anymore. If `ingester.chunk-encoding` is set to `Delta` the ingester will fail to start. #1706
+* [CHANGE] Setting `-ingester.ingester.max-transfer-retries` to 0 now disables hand-over when ingester is shutting down. Previously, zero meant infinite number of attempts. #1771
```

Comment:
A couple of little things:
1. The PR number should be `1777`
2. `ingester.ingester.max-transfer-retries` > `ingester.max-transfer-retries`

Reply:
1. It's issue number
2. Thanks! Fixed.